frontend/src/components/samples/SampleEditContent.js

```diff
@@ -88,6 +89,8 @@ const SampleEditContent = ({ sample, isAdding}) => {
 
   const [form] = Form.useForm()
   const [formErrors, setFormErrors] = useState({})
+  const [disabledQualityFlag, setDisabledQualityFlag] = useState(true)
+  const [disabledQuantityFlag, setDisabledQuantityFlag] = useState(true)
```

Comment:
Do you really need to use useState 🤔 
Especially considering that `setDisabledQualityFlag` and `setDisabledQuantityFlag` seem to be called only once in the component, and they are only called with `sample` variable that's already passed as a prop to the component.